kitsune/customercare/zendesk.py

```diff
@@ -18,6 +18,14 @@ def __init__(self, **kwargs):
         self.client = Zenpy(**creds)
 
     def _user_to_zendesk_user(self, user, email):
+        """Given a Django user, return a Zendesk user."""
+        # If the user already exists in Zendesk return
+        # them instead of creating a new user
+        if self.get_user_by_email(email):
+            return self.get_user_by_email(email)
```

Comment:
Does this match what we return through `ZendeskUser`? If not, we can end up having this helper method returning two different types of ZD users.

Reply:
This returns an actual ZD user, with whatever attributes they have in ZD. Since the goal of this PR is to _not_ update existing users it doesn't really matter if it is exactly the same, since its only purpose is to have a an id to log a ticket against.

Reply:
Let me be specific - if you have a ZD account, we don't want to do anything with that account. 
I don't see a reason to run updates on any of the other user fields like we do when we match a SUMO user to a ZD user - in this case you are anonymous to SUMO so we just care if you exist in ZD, and we want to leave you be because without auth at SUMO we can't be sure you are you. At least, that's my thought process.